UnitTests/MemoryStreamUnitTests/CanWrite.cs

```diff
@@ -51,5 +51,19 @@ public void CanWrite_Byte_Ctor()
                  OutputHelper.WriteLine($"Unexpected exception {ex}");
             }
         }
+
+        [TestMethod]
+        public void CanWrite_bool_false_Ctor()
```

Comment:
great! Can you also add a test that will test you can indeed write when you can? Just in case !

Reply:
There is one for the other constructor, I'll go ahead and add one for the new constructor, best to be thorough.